src/http.ts

```diff
@@ -27,10 +27,12 @@ class Http {
     method,
     path,
     data,
+    host = undefined,
   }: {
     method: HttpMethod;
     path: string;
     data?: any;
+    host?: string | undefined;
```

Comment:
I think this can just be `host?: string;`, then you don't need the default argument above.

Reply:
Done